src/asynk/async_queue.rs

```diff
@@ -613,7 +847,10 @@ mod async_queue_tests {
         let mut connection = pool.get().await.unwrap();
         let transaction = connection.transaction().await.unwrap();
 
-        let mut test = AsyncQueueTest { transaction };
+        let mut test = AsyncQueueTest {
```

Comment:
let's use a builder to create this struct

Reply:
Solve in [e1d31d2](https://github.com/ayrat555/fang/pull/47/commits/e1d31d215656077b385a07f4582a30a6d595a3c5)